carma_wm_ctrl/test/WMBroadcasterTest.cpp

```diff
@@ -2804,12 +2826,6 @@ ASSERT_EQ(value, p1);
   ASSERT_EQ(test_map_elem->speed_limit_.value(), new_speed_limit->speed_limit_.value());
   ASSERT_EQ(test_map_elem->participants_.begin()->data(), new_speed_limit->participants_.begin()->data());
 
-  // Set the map
-  wmb.baseMapCallback(map_msg_ptr);
-
-  sample_proj_string.data = proj_string;
-  wmb.geoReferenceCallback(sample_proj_string);
```

Comment:
This got deleted but doesn't seem to be moved any where. Why delete it?

Reply:
I did not understand what it was trying to test as it did not have any ASSERT calls, but just realized now that if it crashes it is in a way a test as it should not crash. So I added it back with right syntax